paper-input-container.html

```diff
@@ -277,6 +279,20 @@
         @apply(--paper-input-container-input);
       }
 
+	  .input-content ::content input:focus,
```

Comment:
Currently in `this.focused` notifies, but doesn't `reflectToAttribute` until `paper-input`. We don't currently have access to `:host-context([focused])`, right? Would this be something that could be added to `_computeInputContentClass`?